README.md

```diff
@@ -59,6 +86,54 @@ Back on the master, to finally register the node:
 Done. Repeat last two pieces to add more nodes. Create new pods from the master (or just docker containers on the minions), and see that the pods are indeed reachable from each other. 
 
 
+##### Detailed operation
+
+To elaborate on the network configuration that openshift-sdn performs in minion
+mode, it makes use of five network devices:
+
+ - br0, an OVS bridge device;
+ - lbr0, a Linux bridge device;
+ - vlinuxbr and vovsbr, two Linux peer virtual Ethernet interfaces; and
+ - vxlan0, the OVS VxLAN device that provides access to remote minions.
+
+On initialization in minion mode, openshift-sdn creates lbr0 and configures
+Docker to use lbr0 as the bridge for containers; creates br0 in OVS; creates
+the vlinuxbr and vovsbr peer interfaces, which provide a point-to-point
+connection for the purpose of moving packets between the regular Linux
+networking stack and OVS; adds vlinuxbr to lbr0 and vovsbr to br0 (on port 9);
+and adds vxlan0 to br0 (on port 10).
+
+As openshift-sdn sees subnets added to and deleted from the registry by the
+master, it adds and deletes OpenFlow rules on br0 to route packets
+appropriately: packets with a destination IP address on the local minion's
+subnet go to vovsbr (port 9 on br0) and thus to vlinuxbr, the local bridge, and
+ultimately the local container; whereas packets with a destination IP address
+on a remote minion's subnet go to vxlan0 (port 10 on br0) and thus out onto the
+network.
+
+To illustrate, suppose we have two containers A and B where the peer virtual
+Ethernet device for container A's eth0 is named vethA and the peer for container
+B's eth0 is named vethB.  (If Docker's use of peer virtual Ethernet devices is
+not already familiar to you, see https://docs.docker.com/articles/networking/
+for details on networking in Docker.)
+
+Now suppose first that container A is on the local host and container B is also
+on the local host.  Then the flow of packets from container A to container B is
+as follows:
+
+eth0 (in A's netns) -> vethA -> lbr0 -> vlinuxbr -> vovsbr -> br0 -> vovsbr ->
+vlinuxbr -> lbr0 -> vethB -> eth0 (in B's netns).
+
+Next suppose instead that container A is on the local host and container B is on
+a remote host on the same container network.  Then the flow of packets from
+container A to container B is as follows:
+
+eth0 (in A's netns) -> vethA -> lbr0 -> vlinuxbr -> vovsbr -> br0 -> vxlan0 ->
+network\* -> vxlan0 -> br0 -> vovsbr -> vlinuxbr -> lbr0 -> vethB -> eth0 (in
+B's netns).
```

Comment:
Can we move this section to another document? This flow is true of the somewhat un-optimized version in master today, but the underlying architecture will change soon. It will be easier to maintain, especially when we start giving other solutions as choice to the user.